README.md

```diff
@@ -8,12 +8,15 @@ OpenAG is an open-source client of the Half-Life promod Adrenaline Gamer, comple
 
 # Building
 ## Windows
-1. Get Visual Studio 2017 and CMake.
-2. `git submodule update --init`
-3. `mkdir build`
-4. `cd build`
-5. `cmake ..`
-6. `cmake --build . --config Release`
+1. Install Visual Studio 2017 from https://my.visualstudio.com/Downloads?q=Visual%20Studio%202017 (make sure you select Visual Studio 2017 Community version 15.9). In the Visual Studio Installer, select Desktop Development for C++ and not anything else.
+2. Install Git from https://git-scm.com/. Make sure you add it to PATH.
+3. Install CMake from https://cmake.org. Make sure you add it to PATH.
+4. Go to where you saved this cloned repo, right click ans select 'Git Bash Here.' In Git, type: `git submodule update --init`
+5. `mkdir build`
+6. `cd build`
+7. `cmake .. -G "Visual Studio 15 2017"`
+8. `cmake --build . --config Release`
+9. The CMake program will generate the OpenAG.sln file that you can use with Visual Studio 2017. Make sure you DO NOT build using VS2017; save the files you have changed and run the `cmake --build . --config Release` command. The built DLL will be in the Release folder.
```

Comment:
Huh, what's wrong with building using VS?

Reply:
Damn, only just now I found out you could build using VS too, I'll make the changes right away

Reply:
I've made the changes requested, should be fine now :)